docs/getting-started/config-env.md

```diff
@@ -1,23 +1,32 @@
 ---
-title: Configure Environment Variables
+title: Configuring Environment Variables
 ---
 
-# Configure environment variables/secrets
+# Configuring environment variables and secret keys
 
-Take a look at `Envfile`. This file lists all the `ENV` variables we use and provides a fake default for any missing keys. You'll need to get your own free [Algolia credentials](/backend/algolia) to get your development environment running.
+Take a look at `Envfile`. This file lists all the `ENV` variables we use and provides a fake default for any missing keys. You'll need to get your own free [Algolia credentials][agolia] to get your development environment running.
```

Comment:
there's a typo in the name of the anchor, `algolia` instead of `agolia`

Reply:
Great catch😅

Reply:
ahhaha I think you changed both to `agolia` but the name of the service is `Algolia`, with the `L` :-)